src/registrar/utility/errors.py

```diff
@@ -93,6 +95,7 @@ class NameserverError(Exception):
         NameserverErrorCodes.TOO_MANY_HOSTS: ("Too many hosts provided, you may not have more than 13 nameservers."),
         NameserverErrorCodes.MISSING_HOST: ("Name server must be provided to enter IP address."),
         NameserverErrorCodes.INVALID_HOST: ("Enter a name server in the required format, like ns1.example.com"),
+        NameserverErrorCodes.DUPLICATE_HOST: ("Remove duplicate entry"),
```

Comment:
**question (non-blocking)**: Do we have a "content design" review for language like this? @michelle-rago do you know if we have a document where these error messages get sourced from and where we document them?

Reply:
@neilmb We're reviewing error/success messages. That content is in a spreadsheet which makes it much easier to review! https://drive.google.com/drive/folders/1jnRIBcr1ZhI0BFq0i6l21X0lIOtUCIPD

Reply:
@neilmb and this language was in the ticket